app/views/items/show.html.erb

```diff
@@ -1,3 +1,8 @@
 <div class="page-title"><%= @item.name %></div>
-<%= @item.description %>
-<%= @item.unit_price %>
\ No newline at end of file
+<p>Description: <%= @item.description %></p>
+<p>Unit Price: <%= @item.unit_price %></p>
+<%= link_to "Edit #{@item.name}", edit_merchant_item_path(@merchant, @item) %>
+<div class="flash-message">
+<% if flash[:notice] %>
+  <p><%= flash[:notice] %></p>
+<% end %>
```

Comment:
Looks good!

Just a note for later refactoring - we can have an each statement in the app/views/layouts/application file that will take care of displaying flash messages on all pages.

Reply:
Awesome this is exactly what I was wondering. Thank you for the feedback!